secure_redis/tests/test_secure_redis.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import unicode_literals
+
+import django.test
+from django.core import cache
+
+import mock
+
+from .. import client
+
+
+class SecureRedisTestCase(django.test.TestCase):
+    def test_incr(self):
+        redis_cache = cache.caches['default']
+        with self.assertRaises(client.UnsupportedOperation):
+            redis_cache.incr('num1')
+
+    @mock.patch('redis.client.StrictRedis.set')
+    @mock.patch('redis.client.StrictRedis.get')
+    def test_enc(self, get_method, set_method):
+        original_val = "123"
+        set_method.return_value = True
+        redis_cache = cache.caches['default']
+        redis_cache.set('val', original_val)
+        set_method.assert_called_once()
+
+        encVal = set_method.call_args[0][1]
+        self.assertNotEqual(encVal, original_val)
```

Comment:
This makes sense, and looks to be a good way to test that the encryption is happening indirectly. it would be good to also have a test that mocks the encryption method and make sure *that* is being called with the appropriate value.